src/DssSpell.t.sol

```diff
@@ -683,18 +693,18 @@ contract DssSpellTest is DssSpellTestBase {
 
         hevm.warp(SEP_28_2022 + 731 days + 10 days);
         vest.vest(28);
-        assertEq(gov.balanceOf(SNE_WALLET), prevBalance + 540.00 ether);
+        assertEq(gov.balanceOf(SNE_WALLET), prevBalance + 540.00 ether); 
```

Comment:
unnecessary extra space added here.

Reply:
Sorry this is my IDE / dev environment, fixed it.